src/components/TopBar/components/SearchField/tests/SearchField.test.tsx

```diff
@@ -131,6 +131,16 @@ describe('<TextField />', () => {
     });
   });
 
+  it('showFocusBorder', () => {
```

Comment:
Typically this would read more like a proper sentence. 
```suggestion
  it('adds a "BackdropShowFocusBorder" class when "showFocusBorder" is passed', () => {
```